moderator/moderate/auth.py

```diff
@@ -43,15 +43,36 @@ def authenticate(self, **kwargs):
         if not user:
             return None
         profile = user.userprofile
+        nda_group = None
 
         try:
             self.mozillian_user = self.mozillians_client.lookup_user({'email': user.email})
         except (BadStatusCode, ResourceDoesNotExist):
             return None
 
+        # Get alternate emais
+        user_email_domains = [user.email.split('@')[1]]
+        for email_resource in self.mozillian_user['alternate_emails']:
+            user_email_domains.append(email_resource['email'].split('@')[1])
+
+        # Check if the user is member of the NDA group on each login.
+        # Automatically add users with @mozilla* email in the nda group.
+        if not any(email_domain for email_domain in user_email_domains
```

Comment:
``any`` returns ``True`` if at least one item of the iterable is ``True``. Here you don't need to cast to Boolean. It's the same with the following:

```
if not [email_domain for email_domain in user_email_domains if email_domain in settings.TRUSTED]
```